lib/hawkular/alerts/alerts_api.rb

```diff
@@ -20,6 +20,13 @@ def initialize(entrypoint = 'http://localhost:8080/hawkular/alerts', credentials
       super(entrypoint, credentials, options)
     end
 
+    # Return version and status information for the used version of Hawkular-Alerting
+    # @return [Hash{String=>String}]
+    #         ('Implementation-Version', 'Built-From-Git-SHA1', 'status')
+    def fetch_version_and_status
```

Comment:
as I said, this is *not* a critical issue, but I do think we should start looking how usable and convenient is the client, and the best way to start is putting an effort to make it like that through new code.